content/articles/how-to-create-a-winforms-io-manager-for-removable-drives-in-c#/index.md

```diff
@@ -0,0 +1,425 @@
+﻿We use removable IO devices now and then to transfer files to and from our PCs and to also store files. Sometimes we might need a tool to manage these devices and how they interact with our computers for security, privacy, and other reasons.
+
+In this tutorial, we will create a .NET desktop tool that shall enable us to enable or disable USB ports on our computers and also change the Read/Write permissions of the removable drives. We shall start by going through a detailed guide of creating the UI then we shall work on the code behind the above operations.
+
+### Prerequisites
+To be able to follow through this tutorial, you will need:
+1. [Visual Studio](https://visualstudio.microsoft.com/downloads/) with the .NET framework.
+2. Basic knowledge in C# language and the .NET framework.
```

Comment:
The usual preposition that follows knowledge is `of`. Don't use the prepositions in.